src/components/calcite-pick-list/shared-list-render.tsx

```diff
@@ -1,17 +1,18 @@
 import { Host, h } from "@stencil/core";
 import CalciteScrim from "../utils/CalciteScrim";
 import { VNode } from "@esri/calcite-components/dist/types/stencil.core";
+import { CSS } from "./resources";
 
 const renderScrim = (loading, disabled): VNode => {
   return loading || disabled ? <CalciteScrim loading={loading}></CalciteScrim> : null;
 };
 
 export const List = ({ props, text, ...rest }): VNode => {
-  const { disabled, loading, filterEnabled, dataForFilter, handleFilter } = props;
+  const { disabled, loading, filterEnabled, filterSticky, dataForFilter, handleFilter } = props;
   const { filterPlaceholder } = text;
   return (
     <Host aria-disabled={disabled} aria-busy={loading} {...rest}>
-      <header>
+      <header class={{ [CSS.sticky]: filterSticky }}>
```

Comment:
This class has to go on the header, because of certain limitations of `position: sticky`. One limitation is that the immediate parent is the container that its bound to. The other limitation is it will only move over sibling elements. The filter is the only child of header, which means there's nothing for it to hover over. The `header` tag is the sibling to the items `<slot>` which allows this to function properly.

Reply:
nitpick: Maybe the property should be `headerSticky` or `stickyHeader` then?

Reply:
Could there be anything else in the header at some point? The menu actions are stuck as well so maybe header being sticky makes more sense.

Reply:
If we need the prop, I think making it act on the header makes more sense.

Reply:
I don't think this should be an option.

Reply:
Basically, the filter should just be sticky.

Reply:
Yup. makes sense. The issue says `always` and doesn't mention an option.

Reply:
I'm happy with just the filter being sticky too